app/main.py

```diff
@@ -1 +1,47 @@
-#  write your code here
+class Animal:
+
+    def __init__(self, name, appetite: int, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
+        print(f"Eating {self.appetite} food points...")
+        if self.appetite:
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+
+    def __init__(self, name, is_hungry=True):
+        self.name = name
+        self.is_hungry = is_hungry
```

Comment:
Why do you assign this, it you pass `name` and `is_hungry` to super().__ init__?